### PR TITLE
Add german translations.

### PR DESCRIPTION
**Pull request by mazedlx**

### CODE REVIEW

File: src/de/pagination.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    | Pagination Language Lines
+    |--------------------------------------------------------------------------
+    |
+    | The following language lines are used by the paginator library to build
+    | the simple pagination links. You are free to change them to anything
+    | you want to customize your views to better match your application.
+    |
+ */
+
+    'previous' => '&laquo; Zuürck',
```
Comment (padarom) [inline src/de/pagination.php]: Zurück

File: src/de/pages/info.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    'links' => [
+        'title' => 'Brettspiel-Links',
+
+        'categories' => [
+            'associations' => 'Verbände',
+            'communities' => 'Communities',
+        ],
+
+        'descriptions' => [
+            'boardgamegeek' => 'BoardGameGeek ist die größte Brettspiel-Communit der Welt. Von Scott Alden und Derk Solko im Jahr 2000 gegründet.'
```
Comment (padarom) [inline src/de/pages/info.php]: Community

File: src/de/pages/info.php [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+                'Kann ich ein Brettspiel hinzufügen oder anfordern, das der Seite hinzugefügt werden soll?' => 'Du kannst jedenfalls ein Brettspiel auf der Website hinzufügen, wenn es noch nicht da ist. Benutze bitte [das Kontaktformular](info.contact) um uns mitzuteilen, welches Spiel fehlt.',
+                'Kann ich meine Website / meinen Laden / meine Brettspiel auf dieser Website bewerben?' => 'Wir sind offen für Ihre Werbeanfragen. Du kannst uns für weitere Informationen mit dem [Kontaktformular](info.contact) kontaktieren.'
+            ],
+            'Brettspiel-Hersteller' => [
+                'Würdet ihr unser Spiel auf eurer Website bewerten?' => 'Sicherlich, wenn du uns eine Kopie / einen Prototyp schicken kannst, damit wir das Spiel ausprobieren können, sind wir gerne bereit eine gute Rezension über das Spiel zu schreiben.'
+            ],
+            'Shop-Inhaber' => [
+                'Könntet ihr unseren Shop zu eurere Website hinzfügen.' => 'Natürlich, bitte benutze [das Kontaktformular](info.contact) und sende uns einen Link zu deinem Shop und alle anderen Informationen die wir vielleicht benötigen.'
```
Comment (padarom) [inline src/de/pages/info.php]: "eurer"

File: src/de/pages/info.php [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+Die Website enthält viele Links zu anderen Websites, diese Links sind lediglich informativ und der Website-Betreiber ist nicht verantwortlich für die Inhalte oder die Nutzung, die daraus entstehen könnten.'
+    ],
+
+    'faq' => [
+        'title' => 'Häufig gestellte Fragen',
+        'categories' => [
+            'General' => [
+                'Wann werden die Brettspielpreise aktualisiert??' => 'Unsere Server aktualisieren ständig die Preise auf der Website. Aufgrund der großen Anzahl an Geschäften und Brettspielen, die aktualisiert werden müssen, kann es bis zu 72 Stunden dauern, bis ein Preisupdate registriert wird.',
+                'Kann ich ein Brettspiel hinzufügen oder anfordern, das der Seite hinzugefügt werden soll?' => 'Du kannst jedenfalls ein Brettspiel auf der Website hinzufügen, wenn es noch nicht da ist. Benutze bitte [das Kontaktformular](info.contact) um uns mitzuteilen, welches Spiel fehlt.',
+                'Kann ich meine Website / meinen Laden / meine Brettspiel auf dieser Website bewerben?' => 'Wir sind offen für Ihre Werbeanfragen. Du kannst uns für weitere Informationen mit dem [Kontaktformular](info.contact) kontaktieren.'
```
Comment (padarom) [inline src/de/pages/info.php]: "mein Brettspiel". Second part starts with a formal you and ends with an informal you

File: src/de/common.php [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    'player' => 'Spieler|Spieler',
+    'price' => 'Preis|Preise',
+    'publisher' => 'Herausgeber|Herausgeber',
+    'readmore' => 'mehr',
+    'remove' => 'entfernen',
+    'reset' => 'zurücksetzen',
+    'save' => 'speichern',
+    'search' => 'suchen',
+    'send' => 'senden',
+    'slogan' => 'Die größte euroäische Brettspiel-Datenbank.',
```
Comment (padarom) [inline src/de/common.php]: "europäische"

File: src/de/layout.php [hunk 6]
```diff
@@ -0,0 +1,18 @@
+<?php
+
+return [
+    'cookie' => [
+        'button' => 'Okay, lass uns Brettspiele finden!',
+        'text' => 'Diese Website benützt Cookies um deine Einstellungen zu speichern. Damit können wir dir den Inhalt zeigen, der für dich am interessantesten ist..'
```
Comment (padarom) [inline src/de/layout.php]: Standard High German is "benutzt". "benützt" is only used in southern dialects.

File: src/de/pages/boardgames.php [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+return [
+    'titles' => [
+        'detail' => 'Heute die günstigens Preise für :game',
+        'expansionfor' => 'Erweiterung für :game|Erweiterungen für :game',
+        'pricetable' => 'Online Shops'
+    ],
+
+    'hidedeliverycosts' => 'Lieferkosten ausblenden',
+    'more_stores' => 'Zeige mehr Shops an',
+    'released' => 'veröffnetlicht',
```
Comment (padarom) [inline src/de/pages/boardgames.php]: "veröffentlicht"

File: src/de/pages/boardgames.php [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+    'stock' => 'Lagerstand|Lagerstände',
```
Comment (padarom) [inline src/de/pages/boardgames.php]: You probably mean "Lagerbestand" and "Lagerbestände"? "Lagerbestand" is not in the German dictionary.

File: src/de/pages/info.php [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+
+Die Website enthält viele Links zu anderen Websites, diese Links sind lediglich informativ und der Website-Betreiber ist nicht verantwortlich für die Inhalte oder die Nutzung, die daraus entstehen könnten.'
+    ],
+
+    'faq' => [
+        'title' => 'Häufig gestellte Fragen',
+        'categories' => [
+            'General' => [
+                'Wann werden die Brettspielpreise aktualisiert??' => 'Unsere Server aktualisieren ständig die Preise auf der Website. Aufgrund der großen Anzahl an Geschäften und Brettspielen, die aktualisiert werden müssen, kann es bis zu 72 Stunden dauern, bis ein Preisupdate registriert wird.',
+                'Kann ich ein Brettspiel hinzufügen oder anfordern, das der Seite hinzugefügt werden soll?' => 'Du kannst jedenfalls ein Brettspiel auf der Website hinzufügen, wenn es noch nicht da ist. Benutze bitte [das Kontaktformular](info.contact) um uns mitzuteilen, welches Spiel fehlt.',
```
Comment (padarom) [inline src/de/pages/info.php]: "jedenfalls" translates to "anyhow", I don't think it is correct in this context.

File: src/de/passwords.php [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+    |--------------------------------------------------------------------------
+    |
+    | The following language lines are the default lines which match reasons
+    | that are given by the password broker for a password update attempt
+    | has failed, such as for an invalid token or invalid new password.
+    |
+ */
+
+    'password' => 'Passwörter müssen mindestens 6 Zeichen lang sein. Passwörter müssen gleich sein.',
+    'reset' => 'Dein Passwort wurde zurück gesetzt!',
```
Comment (padarom) [inline src/de/passwords.php]: "zurückgesetzt"

File: src/de/pages/info.php (hunk 9 above)
Comment (mazedlx) [reply src/de/pages/info.php]: Jedenfalls also translates to "by all means", I'll leave it as is if that's ok with you.

**Comment by jerodev** [comment]
Thank you for the contribution. 
However, @padarom had already translated to German in #7.

**Pull request closed without merging**